data/projects/w/wagmi-dev.yaml

```diff
@@ -1,23 +1,30 @@
 version: 7
-name: wagmi-dev
-display_name: wagmi
+name: wevm
+display_name: wevm
 description: TypeScript tooling for Ethereum
 websites:
   - url: https://wevm.dev
 social:
   twitter:
     - url: https://twitter.com/wevm_dev
 github:
-  - url: https://github.com/wagmi-dev
   - url: https://github.com/wevm
+  - url: https://github.com/wagmi-dev
 npm:
```

Comment:
Alphabetized these to make it easier to see what was missing. If order matters, happy to switch back and append new packages to the end.